components/List/src/TypographyThemer/MDCSelfSizingStereoCellTypographyThemer.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCSelfSizingStereoCell.h"
+#import "MaterialTypographyScheme.h"
+
+#import <Foundation/Foundation.h>
+
+/**
+ The Material Design typography system's themer for instances of
+ MDCSelfSizingStereoCellTypographyThemer.
+ */
+@interface MDCSelfSizingStereoCellTypographyThemer : NSObject
+
+/**
+ Applies a typography scheme's properties to an MDCSelfSizingStereoCell.
+
+ @param typographyScheme The typography scheme to apply to the component instance.
+ @param cell A component instance to which the typography scheme should be applied.
+ */
++ (void)applyTypographyScheme:(id<MDCTypographyScheming>)typographyScheme
+      toActionSheetController:(MDCSelfSizingStereoCell *)cell;
```

Comment:
_tip:_ Rename from ActionSheetController to BaseCell

Reply:
LOL whoops